### PR TITLE
Coding session 22 may 2023

### PR DESCRIPTION
**Pull request by bbelderbos**
can you test this please?

### CODE REVIEW

File: app/crud.py [hunk 1]
```diff
@@ -50,39 +44,38 @@ def create_art_item(db:Session, dataset_id:int, art_item: ItemSchema):
...
     db.delete(_art_item)
     db.commit()
     """
     pass
 
-# update art_item data
-def update_art_item(db:Session,
-                    dataset_id:int,
+
+def update_art_item(db: Session,
```
Comment (thisAKcode) [inline app/crud.py:62]: here you no longer pass a dataset_id, is it a part of art_item schema or why you skip this parameter?

File: app/crud.py [hunk 2]
```diff
@@ -50,39 +44,38 @@ def create_art_item(db:Session, dataset_id:int, art_item: ItemSchema):
...
                     art_item: ItemSchema):
-    
-    # do i need to do wiht Session as blah blah?
-    artitem = db.query(Item).one_or_none(id=art_item.id)
-    if artitem is None:
-        raise ArtItemNotFound('nothing to update')
-    dataset = db.query(DataSet).one_or_none(dataset_id = dataset_id)
+
+    # one() can raise NoResultFound or MultipleResultsFound
+    _art_item = db.query(Item).one(id=art_item.id)
```
Comment (thisAKcode) [inline app/crud.py:66]: thanks for this one()

**Review by thisAKcode — APPROVED** [review]
thanks. if I do linting myself do I need to run linters each time I push code to remote?

File: app/crud.py (hunk 1 above)
Comment (bbelderbos) [reply app/crud.py:62]: yep you already have it as part of the schema, no?

**Comment by bbelderbos** [comment]
> thanks. if I do linting myself do I need to run linters each time I push code to remote?

I would automate that as much as possible. Here is how:

Increase Python code quality with pre-commit 
https://www.youtube.com/watch?v=XFyLzr5Ehf0

**Pull request merged**